instantsearch-compose/src/main/java/com/algolia/instantsearch/compose/searchbox/internal/SearchBoxComposeImpl.kt

```diff
@@ -1,25 +1,19 @@
 package com.algolia.instantsearch.compose.searchbox.internal
 
-import androidx.compose.runtime.MutableState
-import com.algolia.instantsearch.compose.searchbox.SearchBoxCompose
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.setValue
+import com.algolia.instantsearch.compose.searchbox.SearchQuery
 import com.algolia.instantsearch.core.Callback
 
 /**
- * [SearchBoxCompose] implementation.
+ * [SearchQuery] implementation.
  */
 internal class SearchBoxComposeImpl(
-    override val query: MutableState<String>
-) : SearchBoxCompose {
```

Comment:
Because `SearchBoxCompose`can be misleading, the object here handles only the query, and nothing more.
Also, it is a parameter of a function called `SearchBox`, and having it as a param with that name could be confusing.

Reply:
Makes sene 👍